src/applications/personalization/dashboard/tests/e2e/dashboard.minimal-widgets.cypress.spec.js

```diff
@@ -15,7 +15,23 @@ describe('MyVA Dashboard', () => {
         messaging: false,
         rx: false,
         facilities: [],
-        inProgressForms: [],
+        inProgressForms: [
```

Comment:
This essentially adds a Cypress test to make sure that expired forms in the `GET user/` object are filtered out and not shown on the Dashboard.